ios/BridgeModules/KettleSDKExtension.m

```diff
@@ -5,19 +5,22 @@
 #import <KettleKit/KettleKit.h>
 #import <KettleKit/KettleKit-Swift.h>
 static void InitializeKettle(NSDictionary *launchOptions) {
+  NSString *apiKey = [NSString stringWithFormat:@"%@", KETTLE_API_KEY];
```

Comment:
Extra security to make sure that the KETTLE_API_KEY is present and have a Key on it, not allow empty.